src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        return "Color is " + this.color + ";\n Number is " + this.number + ".";
```

Comment:
```suggestion
        return "Color is " + this.color + "; Number is " + this.number + ".";
```

Reply:
if you want to have it on the separate line then go with System.lineSeparator() instead